### ISSUE TITLE
x/build: adding LUCI builders blocking submit is too manual

### ISSUE THREAD

**Issue by bcmills** [golang/go#65439]
### Go version

N/A

### Output of `go env` in your module/workspace:

```shell
N/A
```

### What did you do?

Review contributed changes that need additional testing (such as running on a `longtest` builder or a particular architecture) before they can be safely merged.

### What did you see happen?

There is a “CHOOSE TRYJOBS” button, but IIRC it doesn't affect auto-submit, and it definitely doesn't stick when a new patchset is uploaded.

It is possible to leave a comment requesting that the author add a `Cq-Include-Trybots:` line to the commit message, but for small externally-contributed changes that can add an extra round-trip.

### What did you expect to see?

Some way that I (as a reviewer) can mark the CL as ok to auto-submit _provided that_ a specific set of additional LUCI builders succeeds.

**Comment by mknyszek** [comment]
Agreed that this is annoying and a regression from previous behavior. I found a relevant upstream issue and posted to it:  https://crbug.com/1498511. This is something we may be able to improve ourselves with some guidance and/or input from the LUCI team.

(Personally, I would really love to also take advantage of the idea in https://crbug.com/1498511 to shrink the Choose Tryjobs list, if possible. I have a planned workaround, but it's kind of a hack.)
